docs/index.asciidoc

```diff
@@ -20,6 +20,7 @@ include::{include_path}/plugin_header.asciidoc[]
 
 ==== Description
 
+Adds arbitrary fields to an event
```

Comment:
Looking at what the plugin does, it seems more like: `Periodically group all events under a certain list of tags into a single event`